packages/driver/src/multi-domain/events/misc.ts

```diff
@@ -30,4 +33,8 @@ export const handleMiscEvents = (Cypress: Cypress.Cypress, cy: $Cy, specBridgeCo
   Cypress.on('url:changed', (url) => {
     specBridgeCommunicator.toPrimary('url:changed', url)
   })
+
+  Cypress.on('multi:domain:visit:url', (args) => {
+    specBridgeCommunicator.toPrimary('visit:url', args)
```

Comment:
could we directly call the spec bridge from navigation?

Reply:
Attached `specBridgeCommunicator` to `Cypress` and updated to call directly in navigation. [5aafb8f](https://github.com/cypress-io/cypress/pull/20467/commits/5aafb8f3a723b6ddea1655fcc032e856ef1737d4)